nbqa/__main__.py

```diff
@@ -452,7 +467,7 @@ def _run_command(
 
 
 def _get_configs(
-    args: argparse.Namespace, kwargs: List[str], tmpdirname: str
+    args: argparse.Namespace, kwargs: List[str], tmpdirname: str, project_root: Path
 ) -> Tuple[bool, bool]:
```

Comment:
The return type of `_get_configs` should be `bool` and not `Tuple[bool, bool]` right?

Reply:
Wow, this surprised me...I guess mypy can't infer types from an argparse namespace. Great catch, thanks!